pkg/k8s/portforward.go

```diff
@@ -162,6 +163,33 @@ func (pf *PortForward) Run() error {
 	return fw.ForwardPorts()
 }
 
+// Init creates and runs a port-forward connection
+func (pf *PortForward) Init(wait ...chan struct{}) error {
+	log.Debug("Starting port forward")
```

Comment:
should these logs include more context like the local and remote addresses of the forward?

Reply:
Sounds like a good idea :+1: